### PR TITLE
[Conversation]: Add ability to add or remove participants when you send a message to the thread conversation

### PR DESCRIPTION
**Pull request by pooja169usp**
# Implementation:
- Added `nylas-contact-search` to show participants in `to` & `cc` fields in the reply section, this allows for adding and removing participants
- Updated Conversation to reflect same internalProps structure as the other components
- Added a new prop `show_dropdown` to `ContactsSearch` component that defaults to `true`. Using this prop to hide the dropdown in Conversation component for now 

# TODO:
- Add a prop to `Conversation` component that will fetch contacts and populate the dropdown suggestions for `nylas-contact-search` in `to` & `cc` fields

[Story details](https://app.shortcut.com/nylas/story/68692)

### CODE REVIEW

**Review by philrenaud — APPROVED** [review]
Works like a charm! The UI of this could use some polish but the functionality is great.

File: components/conversation/src/Conversation.svelte [hunk 1]
```diff
@@ -21,6 +25,7 @@
     Conversation,
     Account,
   } from "@commons/types/Nylas";
+  import "../../contacts-search/src/ContactsSearch.svelte";
```
Comment (arjunkalburgi) [inline components/conversation/src/Conversation.svelte:28]: Is this how we want to nest components? I thought we'd import from npm as a dev dependency?

File: components/conversation/src/Conversation.svelte [hunk 2]
```diff
@@ -133,6 +156,13 @@
 
   let replyBody = "";
 
+  const handleContactsChange =
+    (field: "to" | "from" | "cc") => (data: Participant[]) => {
+      console.log(field, data);
+      reply[field] = data;
+      console.log(reply);
```
Comment (arjunkalburgi) [inline components/conversation/src/Conversation.svelte:163]:
```suggestion
      reply[field] = data;
```

**Review by arjunkalburgi — APPROVED** [review]
🧠 genius feature, couple comments

**Pull request merged**